worker/backend/integration/sample/main.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
(Can't comment on a binary file...)

Noticed the compiled binary is checked in as `worker/backend/integration/sample/sample` - can we `.gitignore` it and build it during testing instead?

Reply:
whoops hehe, totally!

Reply:
rebased w/out the binary and added to gitignore 👍